### PR TITLE
INDY-2154: Cleanup

### PR DESCRIPTION
**Pull request by ArtObr**

### CODE REVIEW

File: sovtoken/sovtoken/test/helper.py [hunk 1]
```diff
@@ -43,7 +44,7 @@ def send_xfer(looper, inputs, outputs, sdk_pool_handle, extra_data=None):
 
 def check_output_val_on_all_nodes(nodes, address, amount):
     for node in nodes:
-        handler = node.write_manager.request_handlers[XFER_PUBLIC][0]
+        handler = next(h for h in node.write_manager.request_handlers[XFER_PUBLIC] if isinstance(h, XferHandler))
```
Comment (anikitinDSR) [inline sovtoken/sovtoken/test/helper.py]: please, use get_handler_by_type_wm helper method from indy-plenum for this

File: sovtoken/sovtoken/test/helpers/helper_node.py [hunk 2]
```diff
@@ -33,17 +37,17 @@ def get_last_ledger_transaction_on_nodes(self, ledger_id):
     @property
     def xfer_handler(self):
         """ Get the xfer request handler of the first node. """
-        return self._nodes[0].write_manager.request_handlers[XFER_PUBLIC][0]
+        return next(h for h in self._nodes[0].write_manager.request_handlers[XFER_PUBLIC] if isinstance(h, XferHandler))
```
Comment (anikitinDSR) [inline sovtoken/sovtoken/test/helpers/helper_node.py]: please, use get_handler_by_type_wm helper method from indy-plenum for this

File: sovtoken/sovtoken/test/helpers/helper_node.py [hunk 3]
```diff
@@ -33,17 +37,17 @@ def get_last_ledger_transaction_on_nodes(self, ledger_id):
...
 
     @property
     def mint_handler(self):
         """ Get the xfer request handler of the first node. """
-        return self._nodes[0].write_manager.request_handlers[MINT_PUBLIC][0]
+        return next(h for h in self._nodes[0].write_manager.request_handlers[MINT_PUBLIC] if isinstance(h, MintHandler))
```
Comment (anikitinDSR) [inline sovtoken/sovtoken/test/helpers/helper_node.py]: please, use get_handler_by_type_wm helper method from indy-plenum for this

File: sovtoken/sovtoken/test/helpers/helper_node.py [hunk 4]
```diff
@@ -33,17 +37,17 @@ def get_last_ledger_transaction_on_nodes(self, ledger_id):
...
 
     @property
     def nym_handler(self):
         """ Get the NYM request handler of the first node. """
-        return self._nodes[0].write_manager.request_handlers[NYM][0]
+        return next(h for h in self._nodes[0].write_manager.request_handlers[NYM] if isinstance(h, NymHandler))
```
Comment (anikitinDSR) [inline sovtoken/sovtoken/test/helpers/helper_node.py]: please, use get_handler_by_type_wm helper method from indy-plenum for this

File: sovtoken/sovtoken/test/helper.py (hunk 1 above)
Comment (ArtObr) [reply sovtoken/sovtoken/test/helper.py]: fixed

File: sovtoken/sovtoken/test/helpers/helper_node.py (hunk 2 above)
Comment (ArtObr) [reply sovtoken/sovtoken/test/helpers/helper_node.py]: fixed

File: sovtoken/sovtoken/test/helpers/helper_node.py (hunk 3 above)
Comment (ArtObr) [reply sovtoken/sovtoken/test/helpers/helper_node.py]: fixed

File: sovtoken/sovtoken/test/helpers/helper_node.py (hunk 4 above)
Comment (ArtObr) [reply sovtoken/sovtoken/test/helpers/helper_node.py]: fixed

File: sovtokenfees/sovtokenfees/test/transactions/test_nym_with_fees.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        utxos=utxos
+    )
+    helpers.sdk.send_and_check_request_objects([req], wallet=sdk_wallet_trustee)
+    request = looper.loop.run_until_complete(
+        build_get_nym_request(sdk_wallet_trustee[1], req.operation['dest']))
+    read_rep = sdk_get_and_check_replies(looper,
+                                         [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_trustee, request)])
+    reply = json.loads(read_rep[0][1]['result'][DATA])
+    assert req.operation['dest'] == reply['dest']
+    assert req.operation['verkey'] == reply['verkey']
```
Comment (ashcherbakov) [inline sovtokenfees/sovtokenfees/test/transactions/test_nym_with_fees.py]: Do we check in this new tests that FEEs were charged appropriately?

Comment (ArtObr) [reply sovtokenfees/sovtokenfees/test/transactions/test_nym_with_fees.py]: sure, fixed

**Pull request merged**